extensions/sql-database-projects/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "sql-database-projects-vscode",
   "displayName": "SQL Database Projects",
   "description": "Enables users to develop and publish database schemas for MSSQL Databases",
-  "version": "1.5.6",

```

Comment:
any reason to bump the version for sqlproj given we are not releasing it this time?